.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 366
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 40
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
I'd rather go with `stale` or something similar. `wontfix` seems like a deliberate choice not to fix the issue, whereas here we're just talking about it most probably being not relevant.

Reply:
Changed to `stale`.